docker-app/qfieldcloud/core/serializers.py

```diff
@@ -535,3 +547,53 @@ class LatestPackageSerializer(serializers.Serializer):
     package_id = serializers.UUIDField()
     packaged_at = serializers.DateTimeField()
     data_last_updated_at = serializers.DateTimeField()
+
+
+class TeamSerializer(serializers.ModelSerializer):
+    organization = serializers.StringRelatedField(source="team_organization")
+    teamname = serializers.CharField(
+        required=True, allow_blank=False, source="username"
+    )
+
+    class Meta:
+        model = Team
+        fields = ("teamname", "organization")
+
+    def to_representation(self, instance):
```

Comment:
```suggestion
    def to_representation(self, instance: Team) -> dict[str, Any]:
```
Final round of typing.